cmd/tailscaled/tailscaled.go

```diff
@@ -96,6 +97,60 @@ var subCommands = map[string]*func([]string) error{
 	"debug":                   &debugModeFunc,
 }
 
+// tryWindowsAppDataMigration attempts to move the Windows state storage
+// directory from its old location to the new location. (Issue 2282)
```

Comment:
change this to 2856 now that I filed #2856 with an overview.

I linked 2282 from that bug.

Also add `Updates #2856` to the commit mesage.

Reply:
Done.